cluster/test-deploy/aws/install-config.yml

```diff
@@ -0,0 +1,26 @@
+baseDomain: ${BASE_DOMAIN}
```

Comment:
This is moving back to the approach we had before #1677, which I think makes sense.  But we may want to put this template in `cluster/test-deploy/openshift/installer/aws.yml` or some such so we can add an `OWNERS` file maintained by `ci-operator/populate-owners.sh`.  If we want to go that way, I can file a separate PR adding `OWNERS` management for `cluster/test-deploy`.

I'm also fine skipping this directory and just using here-docs to generate this YAML.  For example, see [this][1].

[1]: https://github.com/openshift/release/blob/2ee443911599c3f20bf6a4f4f744b4a9917dee8b/ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml#L175-L231

Reply:
I was trying to avoid here-docs, but depending on @petr-muller's response, they may be unavoidable.